content/news/025/index.md

```diff
@@ -496,8 +490,8 @@ has been announced. The NFT will only be acquirable for a limited amount of time
 
 [As promised](https://fishfight.itch.io/ff/devlog/281554/fish-fight-reloaded),
 after months of private prototyping of early invitees, the Fish Fight devs are
-finally ready to make their code and community channels available to the
-general public. They also launched their website!
+finally ready to make their code and community channels available to the general
```

Comment:
FishFight section lacks a one-sentence description of the game. Some of the links - GitHub repo, Discord server, and fishfight.org - should be moved into a braced list there.